scripts/AxionPlotResonances.C

```diff
@@ -0,0 +1,81 @@
+int AxionPlotResonances() {

```

Comment:
This macro should be named `REST_Axion_PlotResonances`. The file should also be renamed the same way. `REST_Axion_PlotResonances.C`. Also have a look to other macros to find out how the documentation should be written.

If you execute `restListMacros` the macro documentation will appear in screen for all the macros available.

Once properly named your macro will be able to execute as follows:

```
restManager PlotResonances
```